src/pudl_rmi/validate.py

```diff
@@ -0,0 +1,178 @@
+"""A module for data validation and QA/QC of the RMI outputs."""
+
+import logging
+from typing import List
+
+import numpy as np
+import pandas as pd
+import sqlalchemy as sa
+
+logger = logging.getLogger(__name__)
+
+
+RMI_TO_PUDL_COLS = {
+    "parent_name": "utility_name_parent",
+    "utility_name": "utility_name_ferc1",
+    "respondent_id": "utility_id_ferc1",
+    "year": "report_year",
+    "FERC_class": "ferc_acct_name",
+    "original_cost": "plant_balance_w_common",
+    "accum_depr": "book_reserve_w_common",
+    "net_plant_balance": "unaccrued_balance_w_common",
+}
+
+RMI_TO_PUDL_FERC_ACCT_NAME = {
+    "other_fossil": "Other",
+    "hydro": "Hydraulic",
+    "regional_transmission_and_market_operation": "Transmission",
```

Comment:
Just based on the snake_case vs. capitalization, it seems like maybe this is PUDL to RMI, and not RMI to PUDL, but I could be wrong.

Reply:
I'm all for snake-casing any column names we have control over.

Reply:
yes. this is in pudl, so changing it all in one place is not available. I'll make an issue for it.